tests/sanity/ignore-2.10.txt

```diff
@@ -1,6 +1,3 @@
-# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
-# SPDX-License-Identifier: GPL-3.0-or-later
-
```

Comment:
REUSE also supports creating a standalone `/path/to/file`.license file with just `SPDX-License-Identifier: GPL-3.0-or-later` (no comment needed). See `reuse addheader --explict` and https://reuse.software/spec/#comment-headers.

Reply:
I've read about it, but that looks a lot more ugly to me :-)